packages/e2e/src/load-test-scheduler.ts

```diff
@@ -0,0 +1,30 @@
+import { Logger } from 'ts-log';
+import { Observable, map, mergeMap, take, timer } from 'rxjs';
+
+export interface LoadTestSchedulerProps<T> {
+  duration: number;
+  callsPerDuration: number;
+  idOffset?: number;
+  callUnderTest: (id: number) => Observable<T>;
+}
+
+export interface LoadTestSchedulerDependencies {
+  logger: Logger;
+}
+
+export const getLoadTestScheduler = <T>(
+  { duration, callsPerDuration, idOffset = 0, callUnderTest }: LoadTestSchedulerProps<T>,
+  { logger }: LoadTestSchedulerDependencies
+): Observable<T> =>
+  new Observable<T>((observer) => {
+    const callIntervalMs = (duration * 1000) / callsPerDuration;
+    logger.info(`Running ${callsPerDuration} calls in ${duration}s, 1 call / ${callIntervalMs}ms`);
```

Comment:
REVIEW: This is meant to be printed during the test. Is there a better way of printing it?

Reply:
Temporary used `logger = console` for the test